configs/caas/cromwell.conf.ctmpl

```diff
@@ -158,6 +162,26 @@ system {
     jobs = 50
     per = 1 second
   }
+
+  # More info: https://docs.google.com/presentation/d/14VgNBDE8Don_oeOOPIWtJQC5OUrcpk3Vip5AQBBCSs4/edit
+  hog-safety {
+    # Have seen a comment referring briefly to about "4k ips per person"(???).
+    # Meanwhile, this config file sets "concurrent-job-limit = 75000".
+    # Based on the calculations in the presentation above: 75k / 4k ~= 19
+    # If you know any additional information about the IP limit, etc., please update this comment.
+    hog-factor = 19
+
+    # This is the name of the workflow option that will be sent via workflow the options file and used for hog factor
+    #calculations.
+    workflow-option = "google_project"
+
+    # Interval between token logs in seconds.
+    token-log-interval-seconds = 300
+  }
+
+  # Cache file hashes within the scope of a root workflow to prevent repeatedly requesting the
+  # hashes of the same files multiple times.
+  file-hash-cache = true
```

Comment:
Does this have any other options (like cache size?)

Reply:
The other parameters do not currently have knobs.